model/vo/TemplateVO.php

```diff
@@ -215,6 +219,30 @@ public function setTaskStoryId($taskStoryId) {
         $this->taskStoryId = $taskStoryId;
     }
 
+    public function getInitTime(): ?int {
+        return $this->initTime;
+    }
+
+    public function setInitTime(?int $initTime) {
+        $this->initTime = $initTime;
+    }
+
+    public function setInitTimeRaw(?string $initTime) {
+        $this->initTimeRaw = $initTime;
+    }
+
+    public function getEndTime(): ?int {
+        return $this->endTime;
+    }
+
+    public function setEndTime(?int $endTime) {
+        $this->endTime = $endTime;
+    }
+
+    public function setEndTimeRaw(?string $endTime) {
+        $this->endTimeRaw = $endTime;
+    }
+
```

Comment:
Where do you use the `set*TimeRaw` variations? I can see you use the getters to build the XML but I can't find where these are set.

Reply:
replied below https://github.com/Igalia/phpreport/pull/505#pullrequestreview-722419466